cypress-io/cypress
# chore: bump rollup & some vite deps

- Closes https://github.com/cypress-io/cypress/pull/25140

### User facing changelog


N/A

### Additional details


Update some deps. For some reason I could not do `yarn cypress:open --component` in `packages/frontend-shared` - I'd get an error:

![image](https://user-images.githubusercontent.com/19196536/207474102-3f662853-0f09-4392-b4a4-e9a26e71b1dc.png)

This was introduced by https://github.com/cypress-io/cypress/pull/25101. Bumping some related deps fixed it.

This problem was not manifesting in some other projects I tested. I do not think it is impacting users, just developers on the Cypress monorepo.

### Steps to test


- Pull down
- Go in packages/frontend-shared
- Open mode
- I used `Card.cy.tsx`
- Usually workflow should work, no errors etc.

### How has the user experience changed?


Works as expected

### PR Tasks


- [ na] Have tests been added/updated?
- [ na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
